SPEC.md

```diff
@@ -177,7 +178,7 @@ Example `payload`:
 ```
 {
   "schemaVersion": 2,
-  "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
```

Comment:
I suspect this gets dropped: https://github.com/google/go-containerregistry/blob/9cf3ed4ac182c640e59a82343512e4c2cacb6d68/pkg/v1/mutate/image.go#L132-L140

(There's also no way to set the config file mediaType... we should either add that to ggcr or infer it like we do the top-level mediaType.)

Reply:
We aren't using an image index, just a manifest. The term "index" has been used to describe the manifest which contains the signatures, but it's not a manifest list

> There's also no way to set the config file mediaType... we should either add that to ggcr or infer it like we do the top-level mediaType.

SGTM

Reply:
never mind, I see what you mean now

Reply:
what I did seems to work fine:
```
base = mutate.MediaType(empty.Image, types.OCIManifestSchema1)
m, _ := base.Manifest()
m.Config.MediaType = types.OCIConfigJSON
```